File: Examenes/JAVA/ProjectSoftModel/src/pe/edu/pucp/projectsoft/model/Proyecto.java
package pe.edu.pucp.projectsoft.model;
import java.util.ArrayList;
public class Proyecto {
    private int idProyecto;
    private Area area;
    private Estudiante estudiante;
    private ArrayList<Docente> jurados;
    private String titulo;
    private String descripcion;
    private byte[] foto;
    private byte[] archivoTemaTesis;
    private boolean activo;

    public int getIdProyecto() {
        return idProyecto;
    }

    public void setIdProyecto(int idProyecto) {
        this.idProyecto = idProyecto;
    }

    public Area getArea() {
        return area;
    }

    public void setArea(Area area) {
        this.area = area;
    }

    public Estudiante getEstudiante() {
        return estudiante;
    }

    public void setEstudiante(Estudiante estudiante) {
        this.estudiante = estudiante;
    }

    public ArrayList<Docente> getJurados() {
        return jurados;
    }

    public void setJurados(ArrayList<Docente> jurados) {
        this.jurados = jurados;
    }

    public String getTitulo() {
        return titulo;
    }

    public void setTitulo(String titulo) {
        this.titulo = titulo;
    }

    public String getDescripcion() {
        return descripcion;
    }

    public void setDescripcion(String descripcion) {
        this.descripcion = descripcion;
    }

    public byte[] getFoto() {
        return foto;
    }

    public void setFoto(byte[] foto) {
        this.foto = foto;
    }

    public byte[] getArchivoTemaTesis() {
        return archivoTemaTesis;
    }

    public void setArchivoTemaTesis(byte[] archivoTemaTesis) {
        this.archivoTemaTesis = archivoTemaTesis;
    }

    public boolean isActivo() {
        return activo;
    }

    public void setActivo(boolean activo) {
        this.activo = activo;
    }
    
}

File: Examenes/22-2/CSharp/ProjectSoft/ProjectSoftModel/Proyecto.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ProjectSoftModel
{
    public class Proyecto
    {
        private int _idProyecto;
        private Area _area;
        private Estudiante _estudiante;
        private BindingList<Docente> _jurados;
        private string _titulo;
        private string _descripcion;
        private byte[] _foto;
        private byte[] _archivoTemaTesis;
        private bool _activo;
        public int IdProyecto { get => _idProyecto; set => _idProyecto = value; }
        public Area Area { get => _area; set => _area = value; }
        public Estudiante Estudiante { get => _estudiante; set => _estudiante = value; }
        public string Titulo { get => _titulo; set => _titulo = value; }
        public string Descripcion { get => _descripcion; set => _descripcion = value; }
        public byte[] Foto { get => _foto; set => _foto = value; }
        public byte[] ArchivoTemaTesis { get => _archivoTemaTesis; set => _archivoTemaTesis = value; }
        public bool Activo { get => _activo; set => _activo = value; }
        public BindingList<Docente> Jurados { get => _jurados; set => _jurados = value; }
    }
}